app/src/helpers/inferFlash.ts

```diff
@@ -38,52 +41,51 @@ function findSync(pattern, basePath, findDir) {
         return;
       }
 
-      if (!findDir) {
+      if (!limitSearchToDirectories) {
         matches.push(childPath);
         return;
       }
 
       if (childIsDirectory) {
         matches.push(childPath);
       }
-    });
+    }
   })(basePath);
   return matches;
 }
 
-function linuxMatch() {
+function findFlashOnLinux() {
   return findSync(/libpepflashplayer\.so/, '/opt/google/chrome')[0];
 }
 
-function windowsMatch() {
+function findFlashOnWindows() {
   return findSync(
     /pepflashplayer\.dll/,
     'C:\\Program Files (x86)\\Google\\Chrome',
   )[0];
 }
 
-function darwinMatch() {
+function findFlashOnMac() {
```

Comment:
Seems like end-of-life for Flash is planned for end of 2020, so officially it is still "supported" until then:

- https://theblog.adobe.com/adobe-flash-update/
- https://blog.mozilla.org/futurereleases/2017/07/25/firefox-roadmap-flash-end-life/
- https://www.blog.google/products/chrome/saying-goodbye-flash-chrome/

Reply:
> Not related to this PR, but I'm not sure how well flash supports anyway. It's difficult to test, and it might be worth considering removing it :p

@jiahaog thinking the same yeah, not even sure it was still working before this PR, and still is after. Oh well, the internet will tell us?

> Seems like end-of-life for Flash is planned for end of 2020

@remusao thx for the precision.